data/common.yaml

```diff
@@ -1,12 +1,12 @@
 ---
 puppetwebhook::pkg_provider: 'puppet_gem'
 puppetwebhook::server_cfg:
-  server_type: 'daemon'
+  server_type: 'simple'
   host: '0.0.0.0'
   logfile: '/var/log/webhook.log'
   loglevel: 'WARN'
   pidfile: '/var/run/puppet_webhook.pid'
```

Comment:
This is now unused, right?

Reply:
no, that is the actual app setting. The systemd service file is reading from the config file, so that has to be set.

Reply:
Fair enough, though I'd have a look at changing the app to make it optional since systemd doesn't need it.